turbinia/workers/psort.py

```diff
@@ -24,7 +24,41 @@
 
 
 class PsortTask(TurbiniaTask):
-  """Task to run Psort to generate CSV output from plaso storage files."""
+  """Builds a typical plaso command, contains logic specific to psort.
+
+    Args:
+      base_command (str): Command to invoke log2timeline (e.g. log2timeline.py)
+      conf (dict): Dynamic config containing the parameters for the command.
+
+    Returns:
+      String for valid psort command.
+    """
+
+  task_config = {
+      'status_view': 'none',
+      'additional_fields': 'yara_match',
+      'output_format': None,
+      'profilers': None,
+  }
+
+  def build_plaso_command(self, base_command, conf):
```

Comment:
I haven't looked too closely to see if these are actually the same between the `PlasoTask` and this version, but maybe we should have the `PsortTask` object extend the `PlasoTask` object to pick up this method?  Off the top of my head, I think the other things in this object (`run()`, `task_config`, etc) would be overridden.

Reply:
Ping.